### DISCUSSION TITLE
Empty Columns on Importing with CSVs

### DISCUSSION THREAD

**Discussion by induviduality** [AntlerVC/firetable#1076] [Support / Q&A] [answered]
I'm trying to import data from a CSV file. On Previewing the data, it seems like all the columns are correctly read.

![image](https://user-images.githubusercontent.com/53386868/211985712-40d66218-7445-4c88-b88e-00e4838ddcc3.png)

However, after importing the data, some of the columns are blank. How do I resolve this?
Here is the dataset I'm using, for reference: https://opendata.arcgis.com/datasets/726277c507ef4914b0aec3cbcfcbfafc_0.csv
![image](https://user-images.githubusercontent.com/53386868/211986195-a985fb9d-98a0-4459-88c6-21fbba26e3e0.png)

**Comment by harinij** [comment]
Hi @induviduality just checking, while importing did you use the Document ids to be autogenerated one or pick a column (and if so which column)?
![Screen Shot 2023-01-13 at 12 15 48 am](https://user-images.githubusercontent.com/307298/212076070-a59b1190-2103-4f23-8520-9ef6c4c8f0c0.png)

**Reply by induviduality** [reply]
I picked the column named 'ID' in the dataset as the ID Column. One more thing I did was that I did not import the FID column in that dataset.

I tried deleting everything and importing the file multiple times, and the exact same columns are left blank once the import is done.

**Comment by iamanishroy** [comment, marked as answer]
Hi @induviduality,
I tried importing on my end and it worked fine.

![image](https://user-images.githubusercontent.com/62830866/212914914-e6e37f84-fd19-48ce-b014-579fc1e8a133.png)
 
Although I discovered a few empty cells in those columns, this was due to the CSV itself not having it.

![image](https://user-images.githubusercontent.com/62830866/212915532-baab290f-6abb-4947-80fe-b9eb7af7afa4.png)

I believe you will find some if you scroll down and look.

**Reply by induviduality** [reply]
Oh wow, I totally missed that. Thank you so much!
Sorry for the inconvenience caused

**Reply by iamanishroy** [reply]
Not a problem.
